### PR TITLE
4910 update docs for authorizedServices and availableServices

### PR DESCRIPTION
**Pull request by cadibemma**
## Summary

- Add description to API to explain the purpose of authorizedServices and availableServices objects

## Related issue(s)

- department-of-veterans-affairs/va-mobile-app#4910

## Testing done

- None

## What areas of the site does it impact?

- None

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

**Review by LindseySaari — DISMISSED** [review]

File: modules/mobile/docs/schemas/User.yml [hunk 1]
```diff
@@ -62,6 +62,7 @@ properties:
               - DSL
               - MHV
       authorizedServices:
+        description: Services that users are allowed to view.
```
Comment (kpethtel) [inline modules/mobile/docs/schemas/User.yml:65]: Could we change this to something like `Services the user is allowed to access. See availableServices for a list of all services available via the mobile API.`?

File: modules/mobile/docs/openapi.yaml [hunk 2]
```diff
@@ -3439,6 +3439,7 @@ paths:
                   meta:
                     properties:
                       availableServices:
+                        description: Services that the app has through MobileAPI.
```
Comment (kpethtel) [inline modules/mobile/docs/openapi.yaml:3442]: Can we change this to something like `All services available through the mobile API. See authorizedServices for a list of services available to the user.`

Comment (cadibemma) [reply modules/mobile/docs/openapi.yaml]: updated!

**Review by kpethtel — APPROVED** [review]
👍

**Pull request merged**